pkg/iac/scanners/terraform/ignore_test.go

```diff
@@ -433,7 +433,7 @@ resource "bad" "my-rule" {
 		{
 			name: "ignore by dynamic block value",
 			inputOptions: `
-// trivy:ignore:*[secure_settings.enabled=false]
+// trivy:ignore:*[secure_settings.0.enabled=false]
```

Comment:
Just wondering if you don't pass the index in now like so, does it break existing behaviour (backwards compatibility)?

Reply:
Now in Trivy if a resource has multiple blocks, only the first one is used for ignoring. [Kept](https://github.com/aquasecurity/trivy/pull/7205/commits/86e8b29d5d15c8bec427de9535e0051a26b7346f) the previous behaviour for backwards compatibility.